### DISCUSSION TITLE
Feature request: Distinct Node Colors for various Groups Perception

### DISCUSSION THREAD

**Discussion by ygtmesci** [cosmograph-org/cosmos#41] [Q&A] [answered]
Hi there,

Before further explanation, I have to say that this library is a masterpiece really!

It's also great that I can provide group metadata for coloring the groups. However it would be if the nodes could have scalar colors rather than having a continuous between pink and blue. (e.g. for the largest 5 group red, yellow, pink ... and grey for the rest of them)

**Comment by rokotyan** [comment, marked as answer]
@yitopeligo Thanks!

If you're talking about cosmograph.app, that's already possible. You can provide additional columns in metadata and color the nodes based on them, e.g.:

<img width="1101" alt="image" src="https://user-images.githubusercontent.com/755708/224228902-9bf4c19d-e449-4c56-9534-ab04ca002f94.png">

![SCR-20230309-tce](https://user-images.githubusercontent.com/755708/224228759-bb495a61-2369-4c70-81d2-43eef7069c26.png)
